lib/builder/webpack/base.js

```diff
@@ -155,15 +155,14 @@ export default class WebpackBaseConfig {
         test: /\.jsx?$/,
         exclude: (file) => {
           // not exclude files outside node_modules
-          if (/node_modules/.test(file)) {
-            for (const module of [/\.vue\.js/].concat(this.options.build.transpile)) {
-              // item in transpile can be string or regex object
-              if (module.test(file)) {
-                return false
-              }
-            }
-            return true
+          if (!/node_modules/.test(file)) {
+            return false
           }
+
+          // item in transpile can be string or regex object
+          const modulesToTranspile = [/\.vue\.js/].concat(this.options.build.transpile)
+
+          return !modulesToTranspile.some(module => module.test(file))
```

Comment:
@manniL I think we could probably revert this bit too. `Array.some()` will test every single item on the array, whereas the original `for()` loop will instantly return upon failure.

Reply:
Nope! `Array.some` has also an early return 😊 (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some)

Reply:
@manniL had no idea, thanks for the reference :)